<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegistrationOrAuthorization</class>
 <widget class="QMainWindow" name="RegistrationOrAuthorization">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>343</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="HaveAccount">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>161</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>У вас есть аккаунт?</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="RegBox">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>50</y>
      <width>331</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="Reg">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>211</width>
       <height>121</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QLineEdit" name="RegLoginL"/>
      </item>
      <item>
       <widget class="QLineEdit" name="RegPasswordL"/>
      </item>
      <item>
       <widget class="QLineEdit" name="RegEmailL"/>
      </item>
     </layout>
    </widget>
    <widget class="QLabel" name="RegPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="RegEmail">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Email</string>
     </property>
    </widget>
    <widget class="QLabel" name="RegLogin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="AuthBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>50</y>
      <width>301</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QLabel" name="AuthPassword">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="AuthLogin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>63</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
    <widget class="QPushButton" name="Auth">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>110</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>Авторизация</string>
     </property>
    </widget>
    <widget class="QWidget" name="verticalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>209</width>
       <height>91</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLineEdit" name="AuthLoginL"/>
      </item>
      <item>
       <widget class="QLineEdit" name="AuthPasswordL"/>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
